react-components/src/hooks/useClickedNode.tsx

```diff
@@ -66,9 +66,9 @@ export const useClickedNodeData = (): ClickedNodeData | undefined => {
 
     setClickedNodeData({
```

Comment:
I think the type `NodeDataResult` can be moved into this file

Reply:
Moved it here.